airflow/utils/python_virtualenv.py

```diff
@@ -75,7 +85,11 @@ def remove_task_decorator(python_source: str, task_decorator_name: str) -> str:
 
 
 def prepare_virtualenv(
-    venv_directory: str, python_bin: str, system_site_packages: bool, requirements: List[str]
+    venv_directory: str,
+    python_bin: str,
+    system_site_packages: bool,
+    requirements: List[str],
+    clone_virtualenv_packages: bool = False,
```

Comment:
I feel we may want to infer this from `system_site_packages` by default. I’d imagine most existing usages of this function specify `system_site_packages` because they want to access Airflow, but that would be broken by this PR since Airflow is no longer a part of the site-packages directory.